test/user-test.js

```diff
@@ -49,12 +51,19 @@ describe('User', function() {
     expect(user.recipesToCook[0].name).to.equal('Chicken Parm');
   });
 
-  it('should be able to filter recipes by type', function() {
+  it('should be able to filter favoriteRecipes or recipesToCook by type', function() {
     user.saveRecipe(recipe);
-    expect(user.filterRecipes('italian')).to.deep.equal([recipe]);
+    user.saveRecipe(recipe2);
+    user.decideToCook(recipe2);
+
+    expect(user.filterRecipes('italian', 'favoriteRecipes')).to.deep.equal([recipe]);
+    expect(user.filterRecipes('mexican', 'recipesToCook')).to.deep.equal([recipe2]);
```

Comment:
I think we should split this into two tests, since we are expecting two separate results here

Reply:
Okay, i'll split these up.